### DISCUSSION TITLE
github2FA

### DISCUSSION THREAD

**Discussion by gzero0115** [community/community#63000] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I have a question about the GitHub 2FA feature. In the image below, it says that when 2FA is enabled within an organization, those who haven't set up 2FA will be removed. Does this mean they will disappear immediately upon enabling   
![IMG_0539](https://github.com/community/community/assets/86992191/9dd0b5ca-7e4d-4908-81c3-c80af2447563)

**Comment by mehulrajdev** [comment]
no, actually they don't.
even I haven't done that

and brother mark this as answer

**Comment by elderguardian** [comment]
Hello,

When an organization enforces 2FA on GitHub, all members without 2FA in the organization have a certain timeframe to enable it before they get removed. The length of the timeframe can be seen by organization administrators when they enable 2FA enforcement.

**Reply by gzero0115** [reply]
If you check the "enable" checkbox, does a window for setting the duration appear?

**Reply by elderguardian** [reply]
I just created a new account and organization to check. I'm sorry, but it turns out my information is outdated, and all members without 2FA will be removed.

![image](https://github.com/community/community/assets/129489839/db6de1bc-b198-4f7c-8e67-e05d2ad2c9b4)

**Comment by Spixmacaw** [comment]
set up your account on GitHub.com to require an authentication code in addition to your password when you sign in.

 2FA initiative is part of a platform-wide effort to secure software development by improving account security for better security 

![](https://user-images.githubusercontent.com/86992191/258787126-9dd0b5ca-7e4d-4908-81c3-c80af2447563.jpeg)
